### ISSUE TITLE
Add email accessibility audits to Governance process with assistance from the ADE team

### ISSUE THREAD

**Issue by artsymartha68** [department-of-veterans-affairs/va.gov-team#110908]
## New initiative
1. **Governance**: Add a new process for email accessibility reviews for all emails that the OCTO products generate. The ADE team is available to assist with this. Reference the backlog ticket for the Governance team, relating to VA Notify and email templates #96390. 
3. **ADE**: Learn what the standards are for accessible emails, and add email accessibility audit to the menu of services that ADE offers. [Reference comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/110908#issuecomment-2923427645) from @coforma-jamie. 
### OCTO OKR

- **OKR 1.1:** Improve satisfaction with our web and mobile products by 5 points. 
- **OKR 1.1 Related Ticket:** #90224

### Background
This recent [report of the inaccessibility of most email ](https://emailmarkup.org/en/reports/accessibility/2025/)is concerning.  

**From the report:**

> #### Government
>
> We found 2556 emails from government domains. 100% failed accessibility tests. 7% with Critical issues and 93% with Serious issues. The most common Critical issue was Images must have alternate text (6.5%), we’re also seeing about 1% with Zooming and scaling must not be disabled (30 emails). As well as issues with allowed ARIA attributes (7 emails) and invalid ARIA attributes (4 emails).
> 
> #### For Serious issues;
> 
> 85.6% Tables used for formatting should have the role attribute
> 83.72% Content inside the body should be wrapped in a dir attribute
> 83.72% Content inside the body should be wrapped in a lang attribute
> Although it’s only a mild issue 49% of emails had Duplicate ID’s

### Impact: Veteran Facing
VA sends a lot of emails to Veterans. We must add this to our design and Governance processes to do the best we can to make sure all emails sent by VA.gov are as accessible as possible. 
- **Brian**, our departing accessibility expert, says this is not easy. Referenced HTML has major issues. Plain text may be issue riddled solution as well. We'll do our best.
- @jeana-adhoc worked with Brian on the zero silent failure emails and VA Notify template engine, and may be able to share her experience with us as well. 

### Resources & Links of interest
- [Email Markup Consortium](https://emailmarkup.org/)
- [Email accessibility checker documentation](https://parcel.io/docs/dev-tools/accessibility-checker)
- [Current ADE menu of services](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/ADE/readme.md#services-we-offer)
- [10 most commonly found accessibility issues in email](https://emailmarkup.org/en/reports/accessibility/2025/#top-10-most-common-accessibility-issues-in-email)
- [Email Marketer’s Guide to Coding for Accessibility](https://www.litmus.com/blog/7-simple-tricks-to-make-your-email-code-more-accessible)
- [Creating Accessible Emails](https://accessibility.huit.harvard.edu/creating-accessible-emails)

**Comment by artsymartha68** [comment]
With advice from @humancompanion-usds, VA Notify owns the template for all the emails that go out. Since ADE is already helping VA Notify for their audit findings, can you add in looking at the templates and the resulting emails?

**Comment by artsymartha68** [comment]
@SarahKay8 just completed an email evaluation via #119270 

Perhaps we can use some of the steps she took for the standard ADE process documentation. I would like to mention deferring to the Content team on any final content decisions since they are always balancing feedback from many stakeholders.
